src/google/protobuf/compiler/objectivec/objectivec_helpers.cc

```diff
@@ -118,15 +118,15 @@ string UnderscoresToCamelCase(const string& input, bool first_capitalized) {
         current = "";
       }
       current += c;  // already lower
-      last_char_was_number = last_char_was_lower = last_char_was_upper = false;
```

Comment:
@thomasvl to take a look at the change to this file.

Reply:
These are fine, the first line in each was clearing all state for consistency, but it is an ignored assignment because of the following lines.